eng/Version.Details.xml

```diff
@@ -31,7 +31,7 @@
     </Dependency>
     <Dependency Name="NuGet.Build.Tasks" Version="5.0.0-preview1.5663">
       <Uri>https://github.com/nuget/nuget.client</Uri>
-      <Sha></Sha>
+      <Sha>cf25cc76caccfa6f68e4c74d7d326fef9f9aa3d4</Sha>
```

Comment:
The version doesn't match what they put in the .props: https://github.com/dotnet/toolset/blob/9b28bd5df9641171622cabc9c4e13e8ad1f181f9/eng/Versions.props#L55

Is this the right SHA for that version? Should we change the Version above to match  the version in props?

Reply:
Ahh sorry I got the sha from a convo but it was for a different train.  Looking it up